Code/client/Services/TransportService.h

```diff
@@ -32,10 +34,12 @@ struct TransportService : Client
 
     [[nodiscard]] bool IsOnline() const noexcept { return m_connected; }
     void SetServerPassword(const std::string& acPassword) noexcept { m_serverPassword = acPassword; }
+    const uint32_t& GetLocalPlayerId() const noexcept { return m_localPlayerId; }
 
 protected:
     // Event handlers
     void HandleUpdate(const UpdateEvent& acEvent) noexcept;
+    void HandleConnect(const ConnectedEvent& acEvent) noexcept;
```

Comment:
We should for safety reasons tombstone the localplayerid on disconnect. e.g. set it to invalid.

Reply:
Fixed with new HandleDisconnected